.github/workflows/deploy-hubs.yaml

```diff
@@ -318,3 +318,24 @@ jobs:
           max_attempts: 3
           command: |
             python deployer run-hub-health-check ${{ matrix.jobs.cluster_name }} ${{ matrix.jobs.hub_name }}
+
+  # This job should run after we deploy staging or prod hubs
+  # It makes sure there is an uptime check and an alerting policy for all our hubs
+  ensure-uptime-checks:
+    runs-on: ubuntu-latest
+    needs: [upgrade-support-and-staging, upgrade-prod-hubs]
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Setup deploy for ${{ matrix.jobs.cluster_name }} cluster
+        uses: ./.github/actions/setup-deploy
+        with:
+          # Use GCP, as we use GCP uptime checks
+          provider: gcp
+          GCP_KMS_DECRYPTOR_KEY: ${{ secrets.GCP_KMS_DECRYPTOR_KEY }}
+
+      - name: ensure uptime checks are set up
+        run: |
+          python3 deployer ensure-uptime-checks
```

Comment:
Hmm, I think that without a cluster passed here, this ensures the checks are setup for the hubs in the 2i2c cluster, for every cluster deployed?

But it's not very clear who `--notification-channel-id` should be? Do we need to create  this channel manually for each gcp project?

Reply:
Perhaps a simpler way to include this would be as a new step in the `upgrade-support-and-staging` and `upgrade-prod-hubs` jobs? They already consume the relevant cluster info and ensure we only run jobs for the clusters/hubs we need to

Reply:
We may also need to maintain a mapping of `cluster name` -> `notification channel ID` somewhere for the script to read, unless we have a catch-all channel set as the default?

Reply:
If we keep is as a separate job, we will need an `if: always()` statement like the above jobs because of the reasoning given in the comment

https://github.com/2i2c-org/infrastructure/blob/c52df2d99fa77c6eb6ffd82e6deda9e09eb89742/.github/workflows/deploy-hubs.yaml#L239-L245

Reply:
I hope https://github.com/2i2c-org/infrastructure/pull/1750/files#diff-f4ac06565f573ca00491b1116529d5b76e373e63f87defa64662235f8e79e739 clarifies this a little bit more. Re: GitHub, it basically needs to run every time at the end if *any* hubs have been deployed I think. I *think* I got it with my new commits, but not 100% sure.